### DISCUSSION TITLE
Vercel tell me that my site used 3931% of your monthly included Serverless Function execution amount

### DISCUSSION THREAD

**Discussion by finalevil** [vercel/community#5179] [Help]
### Question

Hi,

I'm not sure if this question can be posted here, 
but I've been asking the official Vercel support in the support center and haven't received a response.

This is a big problem. 
Our website has used 3931% of the monthly included Serverless Function execution amount, 
resulting in an additional charge of $15,360 on my bill. 

This is about to exceed the cost we can afford.

This status is quite unusual. (see below image)
<img width="948" alt="截圖 2023-12-14 凌晨4 59 28" src="https://github.com/vercel/vercel/assets/2484327/9be18100-5212-45f5-af8a-ff80c1121d9a">

Normal status, about 200 GB-Hrs per day. But now, about 10 TB-Hrs. 
Could someone please assist me in understanding the cause of this problem?
Is someone attacking our website?

Thank you very much.

### Steps to Reproduce

<img width="948" alt="截圖 2023-12-14 凌晨4 59 28" src="https://github.com/vercel/community/assets/2484327/50576f20-f62a-4543-b58e-27c1f0aadac2">

**Comment by leerob** [comment]
Hey there, could you please open a support request and share more information about the project so we can assist? Thank you.

**Reply by finalevil** [reply]
Yes, I have open support request, case number is 00180150
but no one replied.
This is really urgent because the costs are rapidly increasing. 
We hope you can help us check it as soon as possible.

**Reply by amyegan** [reply]
Thanks for your patience. The engineering team is still investigating and will share an update on your support ticket soon.

**Reply by finalevil** [reply]
Could you please explain why the ticket I created has disappeared from the ticket list, and when I directly access the webpage using the case ID, I only see the title but not the content of my ticket?

**Reply by mcsdevv** [reply]
Hi @finalevil, I've taken a look and I'm fairly sure this is an improvement we can make in the UI when a case is merged, we'll get that fixed up!

With regard to your open case, I've provided some further details and questions for you, and I'll be glad to work through this with you to get to a resolution.
